static/usage/datetime/date-constraints/advanced/angular-ts.md

```diff
@@ -0,0 +1,22 @@
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html'
+})
+export class AppComponent {
+  constructor() {}
+  
+  public isWeekday = (dateString: string) => {
```

Comment:
~ Angular's style guide encourages not specifying `public` on methods that are default public: https://angular.io/guide/styleguide#member-sequence (scroll past the incorrect example, to the correct member sequence example).